tests/plugins/module_utils/ca_common.py

```diff
@@ -0,0 +1,150 @@
+#!/usr/bin/python
+
+# -*- coding: utf-8 -*-
+# Copyright (c) 2020 OpenStack Foundation
```

Comment:
so is this a copy from an earlier file? do we need to point to it? do we need/want to add also 2024 here in an earlier line? There are a few more not just this one.

Reply:
Yes, this set of python files represent the core library imported from TripleO: tripleo used to manipulate the ceph spec, which is something required in this context.
I added that library here and extended to fit the use case. I'll fix the header in my next PS

Reply:
I updated the headers at this point, thank you for checking!